Source/Csla.Blazor.Test/ViewModelSaveAsyncErrorTests.cs

```diff
@@ -72,6 +72,51 @@ public async Task SavingSuccess_ErrorEventIsNotInvoked()
       Assert.IsNull(vm.ViewModelErrorText);
     }
 
+    [TestMethod]
+    public async Task SavingSuccess_BusyHelper()
+    {
+      // Arrange
+      IDataPortal<PersonEdit> dataPortal;
+      PersonEdit person;
+
+      // Create an instance of a DataPortal that can be used for instantiating objects
+      dataPortal = _testDIContext.CreateDataPortal<PersonEdit>();
+      person = dataPortal.Create();
+      person.Name = "TestTest";
+      
+      var appCntxt = _testDIContext.CreateTestApplicationContext();
+      var vm = new ViewModel<PersonEdit>(appCntxt)
+      {
+        Model = person
+      };
+
+      await vm.SaveAsync();
+      Assert.IsNull(vm.Exception);
+    }
+
+    [TestMethod]
+    public async Task SavingFailure_ErrorTimeOut()
+    {
+      // Arrange
+      IDataPortal<Person> dataPortal;
+      Person person;
+
+      // Create an instance of a DataPortal that can be used for instantiating objects
+      dataPortal = _testDIContext.CreateDataPortal<Person>();
+      person = dataPortal.Create();
+      person.Name = "TestTest";
+
+      var appCntxt = _testDIContext.CreateTestApplicationContext();
+      var vm = new ViewModel<Person>(appCntxt)
+      {
+        Model = person
+      };
+
+      await vm.SaveAsync();
+      Assert.IsNotNull(vm.Exception);
+      Assert.AreEqual(vm.Exception.Message, "Csla.Blazor.Test.Person.SaveAsync - 00:00:30.");
```

Comment:
This is way too long. We are in a unit test context here and a unit test should complete fast. _Really_ fast. So try to tune it down preferably under 1s and at most 2s.

Reply:
ok. noted.